File: algorithms/insertion_sort/insertion_sort.py
def insertion_sort(arr):
	  for i in range(len(arr)-1):
		    insert(arr, i, arr[i+1])

def insert(arr, sortedIndex, current):
		j = sortedIndex
		while j >= 0 and current < arr[j]:
				arr[j+1] = arr[j]
				j = j - 1
				arr[j+1] = current

File: algorithms/insertion_sort/insertion_sort.js
const insertion_sort = (arr) => {
    for (let i = 0; i < arr.length - 1; i++) {
          insert(arr, i, arr[i+1]);
    }
}

const insert = (arr, sortedIndex, current) => {
      let j = sortedIndex
    for (j; j >= 0 && current < arr[j]; j--) {
          arr[j+1] = arr[j]; 
    }
      arr[j+1] = current;
}